backend/controllers/Task/DestroyController.js

```diff
@@ -13,11 +13,11 @@ class DestroyController {
             return res.sendStatus(this.httpStatusCodes.NO_CONTENT);
         }
 
-        if (task.user_id !== req.session.user.id) {
+        if (task.userId !== req.session.user.id) {
             return res.sendStatus(this.httpStatusCodes.FORBIDDEN);
         }
 
-        await task.destroy();
+        await this.taskService.destroy(task._id, task._rev);
```

Comment:
why have you changed that?

Reply:
task has no `destroy()` method. In couchdb there are no models so we have to destroy via service

Reply:
ok, I forgot that there is no sequelize